### PR TITLE
[DOCS] rewording the installation overview

### PR DESCRIPTION
**Pull request by kblaszczak-intel**
some suggestions for rewording the installation overview

### CODE REVIEW

File: docs/install_guides/installing-openvino-overview.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 
 ## Installation Options
 
-From the 2022.1 release, the OpenVINO installation package has been separated into two parts: OpenVINO Runtime and OpenVINO Development Tools. See the following instructions to decide your installation process.
-
+Since the 2022.1 release, the OpenVINO installation package has been separated into two parts: OpenVINO Runtime and OpenVINO Development Tools. See the following instructions to choose your installation process.
 ### Decide What to Install
 
-**If you have already finished your model development and want to deploy your applications on various devices, [install OpenVINO Runtime](installing-openvino-runtime.md)**, which contains a set of libraries for an easy inference integration into your applications and supports heterogeneous execution across Intel® CPU and Intel® GPU hardware.
+**If you have already finished your model development and want to deploy your applications on various devices, [install OpenVINO Runtime](installing-openvino-runtime.md)**. It contains a set of libraries for easy inference integration with your products.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:14]:
```suggestion
**If you have already finished your model development and want to deploy your applications on various devices, [install OpenVINO Runtime](installing-openvino-runtime.md)**, which contains a set of libraries for easy inference integration with your products.
```

File: docs/install_guides/installing-openvino-overview.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
-**If you want to download model from [Open Model Zoo](../model_zoo.md), convert to [OpenVINO IR](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md), [optimize](../optimization_guide/model_optimization_guide.md) and tune pre-trained deep learning models**, [install OpenVINO Development Tools](installing-model-dev-tools.md), which provides the following tools:
+**If you want to download, convert, optimize, and tune pre-trained deep learning models**, [install OpenVINO Development Tools](installing-model-dev-tools.md), which provides the following tools:
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:16]:
```suggestion
**If you want to download models from [Open Model Zoo](../model_zoo.md), [convert your own models to OpenVINO IR](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) or [optimize and tune pre-trained deep learning models](../optimization_guide/model_optimization_guide.md)**, [install OpenVINO Development Tools](installing-model-dev-tools.md), which provides the following tools:
```

File: docs/install_guides/installing-openvino-overview.md [hunk 3]
```diff
@@ -25,15 +24,15 @@ From the 2022.1 release, the OpenVINO installation package has been separated in
...
+For Python developers, you can [install OpenVINO from PyPI](installing-openvino-pip.md), which contains both OpenVINO Runtime and Development Tools, while requiring fewer steps. 
 
-For C++ developers, you may choose one of the following installation options to install OpenVINO Runtime on your specific operating system:
+For C++ developers, you may choose one of the following installation options for OpenVINO Runtime on your specific operating system:
 
-* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md) or [Docker](installing-openvino-docker-linux.md).
-* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md) or [Docker](installing-openvino-docker-windows.md).
+* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-linux.md).
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:31]:
```suggestion
* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md) or [Docker](installing-openvino-docker-linux.md).
```

File: docs/install_guides/installing-openvino-overview.md [hunk 4]
```diff
@@ -25,15 +24,15 @@ From the 2022.1 release, the OpenVINO installation package has been separated in
...
+* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:32]:
```suggestion
* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md) or [Docker](installing-openvino-docker-windows.md).
```

File: docs/install_guides/installing-openvino-overview.md [hunk 5]
```diff
@@ -25,15 +24,15 @@ From the 2022.1 release, the OpenVINO installation package has been separated in
...
 * macOS: You can install OpenVINO Runtime using an [Installer](installing-openvino-macos.md) or [Anaconda Cloud](installing-openvino-conda.md).
 * [Raspbian OS](installing-openvino-raspbian.md).
 
-> **NOTE**: From the 2022.1 release, OpenVINO Development Tools can **only** be installed via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
+> **NOTE**: With the introduction of the 2022.1 release, the OpenVINO Development Tools can be installed **only** via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-overview.md:36]:
```suggestion
> **NOTE**: From the 2022.1 release, OpenVINO Development Tools can be installed **only** via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
```

**Comment by kblaszczak-intel** [comment]
due to technical reasons, changes could not be made, so an alternative PR was made: 
https://github.com/openvinotoolkit/openvino/pull/11217

**Pull request closed without merging**